sklearn/model_selection/_validation.py

```diff
@@ -1106,6 +1106,14 @@ def learning_curve(estimator, X, y, groups=None,
         Whether to shuffle training data before taking prefixes of it
         based on``train_sizes``.
 
+    stratify : boolean, optional
+        Whether to stratify training data before taking prefixes of it;
+        when True, each training set will be sampled such that the proportion
+        of each class matches the full training data (for the current iteration
+        of cross-validation) as well as possible.
+        Stratification is done based on the y labels.
```

Comment:
Only for classification, right?

Reply:
Currently - yes, until #4757 is resolved. I changed description of this parameter to be consistent with train_test_split for now.